app/javascript/components/App.js

```diff
@@ -14,14 +14,37 @@ import {
 } from 'react-router-dom'
 
 class App extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      apartments: []
+    }
+  }
+
+  componentDidMount(){
+    this.readApartment()
+  }
+
+  readApartment = () => {
+    fetch("http://localhost:3000/apartments")
+    .then(response => response.json())
+    .then(apartmentArray => this.setState({apartments: apartmentArray}))
+    .catch(errors => console.log("Cat read errors:", errors))
+  }
+
+
+
+
+
   render() {
     return (
       
         <Router>
           <Header {...this.props} />
           <Switch>
             <Route exact path="/" component={Home} />
-            <Route path="/apartmentindex" component={ApartmentIndex} />
+            {/* <Route path="/apartmentindex" component={ApartmentIndex} /> */}
```

Comment:
You can remove the static route from your code. It was replaced with the dynamic route you built below.